app/models/message.rb

```diff
@@ -0,0 +1,14 @@
+class Message < ActiveRecord::Base
+  belongs_to :goal
+  belongs_to :user
+
+  validates :content, presence: true
+  delegate :setter_username, to: :goal, prefix: true
+  delegate :title, to: :goal, prefix: true
+  delegate :username, to: :user
+
+  def self.broadcast
+    Message.includes(:user, goal: :setter).order(created_at: :desc).limit(20)
```

Comment:
Changed to "load_news_feed" when it became a before_action, which is more specific.